### DISCUSSION TITLE
Welcome to lora Discussions!

### DISCUSSION THREAD

**Discussion by cloneofsimo** [cloneofsimo/lora#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Suggest next step for LoRA
  * Share your models, workflows, so others can build on them and share even more,
  * Show off works + extensions + anything built on top of this tool.

**Comment by Boldor83** [comment]
i got great results but they are a bit too spicy to post :D thank you for this, i will try some more stuff. The colab strangely didnt't work well but the extension in automatic. The filewords option didn't work, i created txt files along the images but it looks like it didn't read them like for embedding. Does it only work when the words are in the file name itself? i then used a prompt with 3000 steps, 200 for each image on 20 images, i reduced the default learning rate slightly. Took only about 1/2 an hour on 3080 rtx 8 gb vram.
